pliers/extractors/text.py

```diff
@@ -313,3 +335,38 @@ def _extract(self, stim):
         features = ['sentiment_' + k for k in scores.keys()]
         return ExtractorResult([list(scores.values())], stim, self,
                                features=features)
+
+class SpaCyExtractor(TextExtractor):
+    
+    def __init__(self, model='en_core_web_sm', features=None):
+        #verify_dependencies(['spacy'])
+        try: 
+            nlp = spacy.load('en_core_web_sm')
+        except ImportError:
+            logging.warning("Spacy Models ('en_core_web_sm') not found. Downloading and installing")
+            spacy_download("en_core_web_sm-2.0.0/en_core_web_sm-2.0.0.tar.gz")
+           
+            
+            
+    def _extract(self, stim):
+        #make lowercase, remove puntuations:
+        stim=LowerCaseSpecialCharStripExtractor._extract(stim)
```

Comment:
Let's drop this part. It's a good thought, but for now let's leave out any preprocessing. (If the user wants to do some, they can explicitly include a preprocessing transformer in their chain.)